FirebaseRemoteConfig/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 - [fixed] Fixed database creation on tvOS. (#6612)
 - [changed] Updated public API documentation to no longer reference removed APIs. (#6641)
 - [fixed] Updated `activateWithCompletion:` to use completion handler for experiment updates. (#3687)
+- [changed] Log metadata for Remote Config parameter get calls. (#6692)
```

Comment:
Can you make this more descriptive? As written this might raise concerns for developers. What exactly is the impact of the change for users of the RemoteConfig libraries?

Reply:
I think the RC team wants to keep this intentionally vague, since most developers won't be able to use this.

Reply:
In that case, something like "Added preliminary infrastructure to support future features" or nothing at all might be better. If I read the code right, additional logging is only happening when Personalization is being used. Saying unspecified logging is happening can be concerning to developers.

Reply:
cc: @morganchen12

Reply:
Changed to your suggestion.